exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,36 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10){
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isShortName(name) {
+  if (name === "Daniel"){
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
I think the name of the function indicates that you should be checking name length.